src/styles/blocks/page.css

```diff
@@ -0,0 +1,13 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.page__content {
+  margin: 0;
+}
```

Comment:
According to BEM, you could've just left all of this in your index.css file.
Not a bad thing, just pointing it out, less files is better IMO.

Reply:
Yep, for `body` - just reset margin using tag,